webapp/channels/src/components/convert_channel_modal/convert_channel_modal.test.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {shallow} from 'enzyme';
```

Comment:
@sbishel, we are in a process of migrating all enzyme test out since it has been deprecated.  I think we should not add anymore enzyme test into the mix.

Reply:
Thought about that after yesterdays meeting...I will convert.

Reply:
@sbishel i can help, if needed.